.env

```diff
@@ -0,0 +1,6 @@
+FRONTEND_PORT=8080
+BACKEND_PORT=3000
+DATABASE_PORT=5432
+POSTGRES_DB=sk0
+POSTGRES_USER=myuser
+POSTGRES_PASSWORD=myPassword1
```

Comment:
A postgres_user és postgres_password-öt mindenki magának állítja majd be?

Reply:
Since this is locally yes, we totally should.
If we are using docker then we should rather use the ones in the docker compose (or even the .env ones).

Reply:
I'm still confused about docker.  what does local and what does "using docker" mean?